Circular queue implementation in Java

prabhkaur301 opened the pull request:
Circular queue implementation in Java with all necessary functionalities:-
Enqueue()
Dequeue()
isEmpty()
isFull()
printQueue()
showRear()
showFront()

Mewwaa reviewed (commented):
Thanks for your contribution

Mewwaa commented:
Oh I'm so sorry, I can't merge your pr because of CONTRIBUTORS.md. I added your data to it and removed it from here, so if you could just make another pr without that file and I'll merge it

prabhkaur301 commented:
> Oh I'm so sorry, I can't merge your pr because of CONTRIBUTORS.md. I added your data to it and removed it from here, so if you could just make another pr without that file and I'll merge it

Should I delete CONTRIBUTORS.md and then make the PR for this same program? Please clarify.
